.github/workflows/reusable_checks.yml

```diff
@@ -52,6 +52,14 @@ jobs:
     - name: Check Python formatting
       run: cmake --build build --target black-format-check
 
```

Comment:
we can remove `# TODO: add license check` on top of this file now

Reply:
done